### ISSUE TITLE
train loss

### ISSUE THREAD

**Issue by taoxinlily** [javier2491/FaceQnet#10]
Thank you for opensource!
I want to train,but loss did  not become small. So I want to ask which loss when you train?And learning rate?

**Comment by SURABHI-GUPTA** [comment]
hey!
how did you train it from scratch ?

**Comment by javier-hernandezo** [comment]
Hello,

We did not train it from scratch. We took a ResNet-50 model pretrained for face recognition (using VGGFace 2). Then we modify the last layers of that pretrained model and retrained only the new layers using VGGFace 2 with a quality groundtruth we generated (see FaceQnet paper for more details about this).

Regarding training parameters, we used:

Optimizer: Adam (all its parameters by default in Tensorflow)
Loss: MSE
batch size = 128

Hope it will be useful.

Regards.
